dsgrid/cli/registry.py

```diff
@@ -609,9 +609,39 @@ def update_dataset(registry_manager, dataset_config_file, log_message, update_ty
 
 @click.command()
 @click.pass_obj
-def sync(registry_manager):
-    """Sync the official dsgrid registry to the local system."""
+@click.option(
+    "--project_id",
+    "-P",
+    type=str,
+    help="Sync latest dataset(s) version based on Project ID",
+)
+@click.option(
+    "--dataset_id",
+    "-D",
+    type=str,
+    help="Sync latest dataset version based on Dataset ID",
+)
+@click.option(
+    "--version",
+    "-V",
+    type=str,
+    help="Sync a specific dataset version. Must be called with the -D option.",
+)
+def data_sync(registry_manager, project_id, dataset_id):
+    """Sync the official dsgrid registry data to the local system."""
     # aws.sync(REMOTE_REGISTRY, registry_manager.path)
+    # no_prompts = registry_manager.no_prompts
```

Comment:
If you want to get `no_prompts` from the parent command you have to set `@click.pass_context` on the command (probably above `@click.pass_obj on line 611`). Then
```
no_prompts = ctx.parent.params["no_prompts"]
```
Did you already try this?

Reply:
I hadn't gotten there yet but this is helpful! thank you!!